src/screens/OTP/OTP.js

```diff
@@ -79,7 +89,13 @@ class OTP extends React.Component<Props> {
 }
 
 const mapDispatchToProps = (dispatch) => ({
-  confirmOTP: (code: string) => dispatch(confirmOTPAction(code)),
+  confirmOTP: (props: Object, callback: Function) => dispatch(verifyPhoneAction(props, callback)),
```

Comment:
It would be good to type the first argument

Reply:
yeah I can see we have more `Objects` than types, therefore in `verifyPhoneAction` a function, I split it to `{ walletId, phone, oneTimePassword }`

Also, all SDK is untyped, but node-sdk is fully typed as it's TS, a bit of waste. We can use some generator to get all types in `Flow`

Reply:
We need to start to write types and use less `Object` :) I put that kind of comment only for this method as we should avoid having magic function params those are objects and are untyped

Reply:
added

export type VerificationPhoneAction = {
  wallet: string,
  phone: string,
  oneTimePassword: string,
}```